src/axiselements.jl

```diff
@@ -482,18 +482,29 @@ end
 # TableFile #
 #############
 
+"""
+$(TYPEDEF)
+
+Placeholder for a table for which data is read directly from a file. Use the
+[`Table`](@ref) constructor.
+"""
 struct TableFile <: OptionType
     options::OrderedDict{Any, Any}
     path::AbstractString
 end
 
 """
-    TableFile([options], path)
+    $SIGNATURES
 
 For reading tables directly from files. See the `pgfplots` manual for the
 accepted format.
+
+If you don't use an absolute path, it will be converted to one.
 """
-TableFile(path::AbstractString) = TableFile(OrderedDict{Any, Any}(), path)
+Table(options::OrderedDict{Any, Any}, path::AbstractString) =
```

Comment:
It is a tiny bit weird that the `Table` constructor does not return a `Table`. However, in practice, I don't think it matters.

Reply:
This allows dispatching `print_tex` on the type. 

If this was the last issue and you don't mind it this way, can we merge?

Reply:
Sure, go ahead.